### ISSUE TITLE
ToggleSwitch styling breaks on Mobile with long label name

### ISSUE THREAD

**Issue by MattWindle** [themesberg/flowbite-react#884]
## Steps to reproduce

Have a `<ToggleSwitch />`  with a long label name

```
<ToggleSwitch
  label="This is a rather large label which breaks styling "
  checked
/>
```

## Current behavior

The ToggleSwitch is collapsing at smaller resolutions when the label text is quite long. 

![image](https://github.com/themesberg/flowbite-react/assets/75845698/0c3e562c-90f2-4958-8dd5-df449eba0068)

## Expected behavior

I expected the ToggleSwitch to remain styled on smaller devices, even when the label text is quite long.

## Context

In our UI we have the need for some quite long labels but these are breaking the ToggleSwitch Styling as seen above - It looks to be when the text moves onto 2 lines.

**Comment by ryan-walsh-forte** [comment]
This issue doesn't just happen with very long labels.

Visit the docs at https://www.flowbite-react.com/docs/components/forms#toggle-switch and make the screen narrow:

![image](https://github.com/themesberg/flowbite-react/assets/132918682/015cc917-db31-4110-a30b-27fdee9ebe33)

**Comment by SutuSebastian** [comment]
Fixed in [flowbite-react@0.10.0](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.10.0) release.
